packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -354,6 +356,23 @@ class XcodeProjectInterpreter {
   }
 }
 
+/// Environment variables prefixed by FLUTTER_XCODE_ will be passed as build configurations to xcodebuild.
+/// This allows developers to pass arbitrary build settings in without the tool needing to make a flag
+/// for or be aware of each one. This could be used to set code signing build settings in a CI
+/// environment without requiring settings changes in the Xcode project.
+List<String> environmentVariablesAsXcodeBuildSettings() {
```

Comment:
@christopherfujino Is this more or less readable than what I had before?

Reply:
It's glorious.